pkg/odo/cli/init/init.go

```diff
@@ -185,7 +185,9 @@ func (o *InitOptions) run(ctx context.Context) (devfileObj parser.DevfileObj, pa
 		infoOutput = messages.SourceCodeDetected
 	}
 	log.Title(messages.InitializingNewComponent, infoOutput, "odo version: "+version.VERSION)
-	log.Info("\nInteractive mode enabled, please answer the following questions:")
+	if len(o.flags) == 0 {
+		log.Infof("\n%s", messages.InteractiveModeEnabled)
+	}
```

Comment:
Can you add a print statement here printing a new line? Currently the message `Dowloading devfile ...` displayed seems very compact.
![Screenshot from 2022-09-07 11-17-44](https://user-images.githubusercontent.com/11015077/188798384-b283158f-570b-4dc3-a866-5cb9747d5940.png)

Reply:
Or do we want to keep the header for the non-interactive command?

Reply:
I find the header to provide helpful information about what the command is doing, even in non-interactive mode. It is also displayed in other non-interactive commands like `odo deploy`. For consistency, I think I'll keep the header for now, and print a new line before "Downloading devfile ...".

Reply:
Done - see [b48425e91943a6112e904b58eb204d8f2ca7db02](https://github.com/redhat-developer/odo/pull/6100/commits/b48425e91943a6112e904b58eb204d8f2ca7db02).

Reply:
I thought you were going to add a header to `odo init`'s non-interactive mode. Were you?

Reply:
The header is what we already have in both non-interactive and interactive modes: the logo, the "Initializing new component" and "odo version" messages.
Unless I've misunderstood the comments, I kept this header as it is now, but simply added a new line after it.